nbgrader/formgrader/templates/feedback.tpl

```diff
@@ -126,6 +126,9 @@ span.nbgrader-label {
             <li><a href="#comment-{{ cell.metadata.nbgrader.grade_id }}">Comment</a></li>
             {% endif %}
           {% endfor %}
+          {% if resources.nbgrader.late_penalty %}
+            <li style="color: #d2413a;">Late submission penalty (Score: -{{ resources.nbgrader.late_penalty | float | round(2) }})</li>
```

Comment:
This `li` tag should probably be given a class and then define the color using that class in the CSS at the top of the page.